docs/alba.js

```diff
@@ -0,0 +1,133 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+  const lam = 128;
+  const HASH_FACTOR = 1e8;
+
+  const node = document.getElementById('main_chart');
+  const n_p = document.getElementById('n_p');
+  const n_f = document.getElementById('n_f');
+  const item = document.getElementById('item');
+  const proof_size = document.getElementById('proof_size');
+  const s_p = document.getElementById('s_p');
+  const proof_proba = document.getElementById('proof_proba');
+
+  function U(n_p, n_f) {
+    return Math.ceil((lam + Math.log2(lam) + 5 - Math.log2(Math.log2(Math.E))) / Math.log2(n_p / n_f));
+  };
+
+  function probabilityOfProof(u, n_p, n_f, x) {
+    // from https://github.com/cardano-scaling/alba/discussions/17
+
+    S_1_bot = n_p / (17 ** 2 / (9 * Math.log2(Math.E)) * u ** 2) - 7 < 1;
+    S_2_bot = n_p / (17 ** 2 / (9 * Math.log2(Math.E)) * u ** 2) - 2 < 1;
+
+    if (!S_1_bot && !S_2_bot) {
+      throw Error("S_1_bot or S_2_bot");
+    }
+
+    const d = Math.ceil(32 * Math.log(12) * u);
+    const q = 2 * Math.log(12) / d;
+    const r = Math.ceil(lam);
+
+    return r * (x / n_p) ** u * d * q;
```

Comment:
```suggestion
    return Math.min(r * (x / n_p) ** u * d * q, 1);
```
This (expectedly, due to the union bound) returns values larger than 100% for large `n_p`. Alternatively, the plot can be truncated at 100% on the y axis (as in https://github.com/cardano-scaling/alba/discussions/17).